src/pandas_profiling/visualisation/plot.py

```diff
@@ -581,6 +601,50 @@ def plot_acf_pacf(config: Settings, series: pd.Series, figsize: tuple = (15, 5))
     return plot_360_n0sc0pe(config)
 
 
+def _plot_acf_pacf_comparison(config: Settings, series: List[pd.Series], figsize: tuple = (15, 5)) -> str:
+    colors = config.html.style.primary_colors
+    n_labels = len(config.html.style._labels)
+    colors = create_comparison_color_list(config)
+
+    _, axes = plt.subplots(nrows=n_labels, ncols=2, figsize=figsize)
+    for i, serie in enumerate(series):
```

Comment:
```suggestion
    for serie, (acf_axis, pacf_axis), color in zip(series, axes, colors):
```
(and the relevant changes in the loop too)?